apps/judicial-system/web/src/routes/Court/InvestigationRequest/Confirmation/Confirmation.tsx

```diff
@@ -56,7 +61,7 @@ const Confirmation = () => {
         // TODO: Handle error
       }
     } catch (e) {
-      // TODO: Handle error
+      toast.error(formatMessage(errors.requestRulingSignature))
```

Comment:
Consider removing this try / catch as the error is already handled in the useCase hook.